naslib/defaults/trainer.py

```diff
@@ -198,6 +209,12 @@ def search(self, resume_from="", summary_writer=None, after_epoch: Callable[[int
             if after_epoch is not None:
                 after_epoch(e)
 
+        logger.info(f"Saving architectural weight tensors: {self.config.save}/arch_weights.pt")
```

Comment:
Move the saving log message inside the `hasattr(self.config, "save_arch_weights") and self.config.save_arch_weights` if block